frontend/templates/views/partials/version-toggle.j2

```diff
@@ -2,9 +2,13 @@
 {# Since it doesn't use grow, global page context is not available. #}
 <h1>
 {{title}}
+[% if versions and versions.length > 1 %]
 <div class="ap--versions">
   {% for v in versions %}
-    <a{% if v !== version %} href="../{{title}}-v{{v}}.html"{% endif %}>{{v}}</a>
+  [% if versions and versions.includes('{{v}}') %]
```

Comment:
I really don't like that we have the versions in the SSR, but do the rendering of the template on import.
The markdownDocument.insertVersionToggler replaces the title and adds the version toggle. So I see no reason why we should not simply remove the title and render the title (like other pages) in the template and add the version toggle there, using the generated json structure.